collect_app/src/main/java/org/odk/collect/android/adapters/FileArrayAdapter.java

```diff
@@ -31,57 +31,47 @@
 import org.odk.collect.android.logic.DriveListItem;
 
 import java.text.SimpleDateFormat;
-import java.util.Collections;
-import java.util.Comparator;
 import java.util.Date;
 import java.util.List;
 import java.util.Locale;
 
-public class FileArrayAdapter extends RecyclerView.Adapter<FileArrayAdapter.ViewHolder> {
+import io.reactivex.annotations.NonNull;
+
+public class FileArrayAdapter extends ArrayAdapter {
```

Comment:
Please add the generic type: 
`public class FileArrayAdapter extends ArrayAdapter<DriveListItem>`